models/User.js

```diff
@@ -1,5 +1,102 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
+
+//Do we want start dates required?
+
+//Create education schema for use in ResumeSchema
+const EducationSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  program:{
+    type: String,
+    required: true,
+  },
+  grade:{
+    type: Number,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  }
+});
+
+//paid/volunteer experience schema for use in ResumeSchema
+const ExperienceSchema = new Schema({
+  title:{
+    type: String,
+    required: true,
+  },
+  company:{
+    type: String,
+    required: true,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  },
+  description:{
+    type: String
+  }
+});
+
+//Project Schema for use in ResumeSchema
+const ProjectSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  stack:{
```

Comment:
I'm not sure what stack refers to here, could we get some clarification?

Reply:
in this schema, `stack` is referring to the technology stack used for the construction of the project, thanks for pointing that out. I think it would be a good idea to rename this to `tech_stack`.

Reply:
```suggestion
  tech_stack:{
```

Reply:
`tech_stack` sounds good to me